Makefile

```diff
@@ -167,6 +167,20 @@ docker:
 .PHONY: build install update-tools tools godocs clean format lint \
 test-cli test-race test-unit test test-import
 
+test-contract:
+	env GOBIN= go get -u golang.org/x/tools/cmd/stringer
+	env GOBIN= go get -u github.com/kevinburke/go-bindata/go-bindata
+	env GOBIN= go get -u github.com/fjl/gencodec
+	env GOBIN= go get -u github.com/golang/protobuf/protoc-gen-go
+	env GOBIN= go install ./cmd/abigen
```

Comment:
can we move these to `make tools`?

Reply:
you can also use `$(GOBIN)` instead of `env GOBIN=`

Reply:
:+1: will do